pkg/accounts/accounts.go

```diff
@@ -61,3 +65,36 @@ func (ac *Account) Clone() couchdb.Doc {
 func (ac *Account) Valid(field, expected string) bool {
 	return field == "account_type" && expected == ac.AccountType
 }
+
+func init() {
+	hooks.Add(consts.Accounts, hooks.EventDelete,
+		func(i *instance.Instance, doc couchdb.Doc, old couchdb.Doc) error {
+			trigs, err := stack.GetScheduler().GetAll(i.Domain)
+			if err != nil {
+				logger.WithDomain(i.Domain).Error(
+					"Failed to fetch triggers after account deletion: ", err)
+				return err
+			}
+			for _, t := range trigs {
+				var toDelete = false
+
+				var msg struct {
+					Account string `json:"account"`
+				}
+				err := t.Infos().Message.Unmarshal(&msg)
+				if err != nil {
+					toDelete = true
```

Comment:
Are you sure? Won't it delete all the triggers that are not tied to a konnector (sharings, thumbnails, etc.)?